vendor/bat-native-ads/src/bat/ads/internal/base/locale/subdivision_code_util.cc

```diff
@@ -9,9 +9,20 @@
 
 #include "base/check_op.h"
 #include "base/strings/string_split.h"
+#include "bat/ads/internal/geographic/subdivision/supported_subdivision_codes.h"
+#include "brave/components/l10n/common/locale_util.h"
 
 namespace ads::locale {
 
+bool IsSupportedLocaleForSubdivisionTargeting(const std::string& locale) {
```

Comment:
nit: maybe change to IsSupportedCountryCodeForSubdivisionTargeting and pass in `brave_l10n::GetISOCountryCode(locale)`